.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+#### What's this PR do?
+
+
+
+#### Where should the reviewer start?
+
+
+
+#### How should this be manually tested?
+
+
+
+#### Any background context you want to provide?
+
+
+
+#### What are the relevant tickets?
+
+
+
+#### What gif best describes this PR or how it makes you feel?
+
+
+
+#### Definition of Done:
+
+- [ ] Is there appropriate test coverage? (~80% coverage)
+- [ ] Is the README appropriately updated?
+- [ ] Are there breaking changes? Has a representative from each affected team signed-off?
```

Comment:
The wording here is a bit odd for an open source project, maybe drop the second question in favor of a statement such as "Please add notes about what they are and why the change is necessary."?